pkg/cmd/step/scheduler/test_data/step_scheduler_config_apply/gitops_all/config.yaml

```diff
@@ -23,6 +23,8 @@ branch-protection:
 deck:
   spyglass: {}
 gerrit: {}
+github:
+  LinkURL: null
```

Comment:
why is this a capital 'L' in 'LinkURL', that sticks out a fair bit given that our usual naming

Reply:
Is this something new we introduced somewhere?

Reply:
I think it's a serialization change between the Prow version we've been depending on and the Prow version Lighthouse derives from.

Reply:
Regardless, it's from the Prow code we derived from, not something we added for Lighthouse.

Reply:
ok. Really sticks out :-(